pkg/e2e/operator/pod_placement_config_test.go

```diff
@@ -208,6 +208,74 @@ var _ = Describe("The Multiarch Tuning Operator", func() {
 			verifyPodLabels(ns, "app", "test", e2e.Present, schedulingGateLabel)
 		})
 	})
+	Context("The webhook should not gate pods with node selectors that pin them to the control plane", func() {
+		BeforeEach(func() {
+			err := client.Create(ctx, &v1beta1.ClusterPodPlacementConfig{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "cluster",
+				},
+				Spec: v1beta1.ClusterPodPlacementConfigSpec{
+					NamespaceSelector: &metav1.LabelSelector{
+						MatchExpressions: []metav1.LabelSelectorRequirement{
+							{
+								Key:      "multiarch.openshift.io/exclude-pod-placement",
+								Operator: "DoesNotExist",
+							},
+						}}}})
+			Expect(err).NotTo(HaveOccurred())
+			Eventually(deploymentsAreRunning).Should(Succeed())
+		})
+		It("should not gate pods that have the node-role.kubernetes.io/master node selector", func() {
```

Comment:
IIUC we can loop through DescribeTable in this case

```suggestion
		DescribeTable("should not gate pods to schedule in control plane nodes", func(selector string) {
```